src/app/recruitment/[id]/components/SimilarJob.tsx

```
@@ -1,8 +1,24 @@
+import { useRecruitmentDetail } from '@/hooks/queries/useRecruitment';
+
+function getRandomPostingIds(count = 6, min = 30000, max = 35000) {
+  const ids = new Set<number>();
+  while (ids.size < count) {
+    ids.add(Math.floor(Math.random() * (max - min + 1)) + min);
+  }
+  return Array.from(ids);
+}
+
+function useMultipleRecruitmentDetails(ids: number[]) {
+  return ids.map((id) => useRecruitmentDetail({ postingId: id }));
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)